tools/deploy_documentation.sh

```diff
@@ -30,13 +30,13 @@ set -e
 
 # Clone the sources files and po files to $SOURCE_DIR/docs_source
 git clone --depth=1 $SOURCE_REPOSITORY docs_source
-cp -r docs_source/docs/. $SOURCE_DIR/docs/
+rclone sync -v --exclude='locale/**' docs_source/docs docs/
```

Comment:
Rclone has implicit trailing slashes coming from rsync: https://rclone.org/commands/rclone_copy/

```suggestion
rclone sync -v --exclude='locale/**' docs_source/docs docs
```